src/sentry/lang/native/symbolicator.py

```diff
@@ -360,6 +361,18 @@ def _ensure_open(self):
         if not self.session:
             raise RuntimeError("Session not opened")
 
+    def _process_response(self, json):
+        source_names = {source["id"]: source.get("name") for source in self.sources}
+        source_names[INTERNAL_SOURCE_NAME] = "Sentry"
+
+        for module in json.get("modules") or ():
+            for candidate in module.get("candidates") or ():
+                candidate["source_name"] = (
```

Comment:
This is aligned with: 

https://github.com/getsentry/sentry/blob/71fcb6d6165060c06e9d763f868a8c4770eba91b/src%2Fsentry%2Fstatic%2Fsentry%2Fapp%2Ftypes%2FdebugImage.tsx#L62-L67